swiggy-pdp-cart/src/app.js

```diff
@@ -0,0 +1,12 @@
+import Header from "./header";
+import Main from "./main";
+import Footer from "./footer";
+export default function App() {
+  return (
+      <>
```

Comment:
There has to be a wrapper `div` to drive the application view and gutter space in general. This should not be an empty tag